natchez-extras-http4s/src/main/scala/com/ovoenergy/natchez/extras/http4s/client/TracedClient.scala

```diff
@@ -24,7 +25,11 @@ object TracedClient {
   private def trace[F[_]]: F ~> Traced[F, *] =
     Kleisli.liftK
 
-  def apply[F[_]: Sync](client: Client[F], config: Configuration[F]): TracedClient[Traced[F, *]] =
+  def apply[F[_]: Sync](
+    client: Client[F],
+    config: Configuration[F],
+    removeNumericPathSegments: Uri => String = removeNumericPathSegments
```

Comment:
@cosmir17 we should rename the function parameter name to avoid variable shadowing.

Reply:
Thank you for your feedback. I've renamed the apply methods' parameters to avoid variable showing as you suggested.